pkg/server/image_pull.go

```diff
@@ -385,6 +399,13 @@ func (c *criService) registryEndpoints(host string) ([]string, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "get default host")
 	}
+	for i, _ := range endpoints {
```

Comment:
lint complains about this one... suggests:
for i := range..

Reply:
done